cpp/src/arrow/util/bit_util_test.cc

```diff
@@ -154,8 +154,8 @@ void ASSERT_BYTES_EQ(const uint8_t* left, const std::vector<uint8_t>& right) {
   ASSERT_EQ(left_array, right);
 }
 
-TEST(BitUtilTests, TestIsMultipleOf64) {
```

Comment:
Can you undo the test naming changes here?

Reply:
Ah, good one. Fixed.